validphys2/src/validphys/photon/compute.py

```diff
@@ -141,21 +143,20 @@ def compute_photon_array(self, replica):
             # it has to be done inside vp-setupfit
 
             # construct PDFs
-            pdfs_init = np.zeros((len(eko.rotations.inputpids), len(XGRID)))
-            for j, pid in enumerate(eko.rotations.inputpids):
+            pdfs_init = np.zeros((len(eko.bases.inputpids), len(XGRID)))
+            for j, pid in enumerate(eko.bases.inputpids):
                 if pid == 22:
                     pdfs_init[j] = photon_qin
                     ph_id = j
                 else:
                     if pid not in self.luxpdfset.flavors:
                         continue
                     pdfs_init[j] = np.array(
-                        [self.luxpdfset.xfxQ(x, Q_IN, replica, pid) / x for x in XGRID]
+                        [self.luxpdfset.xfxQ(x, self.q_in, replica, pid) / x for x in XGRID]
                     )
 
             # Apply EKO to PDFs
-            q2 = eko.mu2grid[0]
-            with eko.operator(q2) as elem:
+            for (mu2, nf), elem in eko.items():
```

Comment:
This is failing since it is not exactly a dict object: AttributeError: 'EKO' object has no attribute 'values'
@AleCandido how would you write it?

Reply:
`EKO` is not a dict. If you want to iterate the operators, just use `eko.operators`.

However, it is unlikely that you want them without labels, because they are not sorted (and actually shuffled by a hash function, so it's deterministic, but well...).

For the same reason, it's unlikely you don't want to use `(mu2, nf)` info.